control_panel_api/auth0.py

```diff
@@ -45,7 +45,7 @@ def get_group_members(self, app_name):
         return group.get_members()
 
     def add_group_member(self, group_name, email):
-        group = self.authorization_api.get_or_create(Group(name=group_name))
+        group = self.authorization_api.get(Group(name=group_name))
```

Comment:
What about when we genuinely need to create the group the first time?

Reply:
The group is created by the app deployment pipeline

Reply:
But I guess as long as we're all using `slug`, this can remain `get_or_create`